ogr/abstract.py

```diff
@@ -132,11 +132,11 @@ def __init__(self, repo: str, service: GitService, namespace: str) -> None:
         self.repo = repo
         self.namespace = namespace
 
-    def is_forked(self) -> bool:
+    def is_forked(self) -> Optional["GitProject"]:
```

Comment:
The method starting with `is` should return bool. This would be confusing.

Reply:
Well you can always do `if git_project.is_forked()`...

My point here is that I want to get the fored_project as soon as possible: but the truth is that I can call get_fork() instead.

I'll change it.